scst-store/sca_scans/CVE.md

```diff
@@ -0,0 +1,42 @@
+# SCA Scanning Results
+
```

Comment:
Add date?  I know it's in the file name but perhaps good to include it here as well

Reply:
Added